tests.lua

```diff
@@ -66,6 +66,21 @@ function LibSerialize:RunTests()
         assert(tab.nested.c == nil)
     end
 
+    do
```

Comment:
You'll need more test cases for the other scenarios we've brought up, e.g. actually triggering a yield, multiple simultaneous async serializations/deserializations (including mixing those two categories), propagating an error from a failed serialization/deserialization.

Reply:
Try to remove all global states that may share between API calls.
Test for multiple simultaneous async serializations/deserializations is still needed.